plugins/modules/win_domain_computer.ps1

```diff
@@ -146,6 +157,73 @@ Function Add-ConstructedState($desired_state) {
   $result.changed = $true
 }
 
+Function Invoke-OfflineDomainJoin {
+  [CmdletBinding(SupportsShouldProcess=$true)]
+  param(
+    [Parameter(Mandatory=$true)]
+    [System.Collections.IDictionary]
+    $desired_state ,
+
+    [Parameter()]
+    [bool]
+    $ReturnBlob = $false ,
```

Comment:
Any reason why this can't be a `-Switch` to be more powershell-like?

Reply:
No reason; I can change that.

Reply:
I think in this case you should, boolean parameters aren't very common in PowerShell and when they are used they are typically can just be a -Switch.

Reply:
No longer needed as there's no bool/switch param.